### PR TITLE
signin and signup

### PR DESCRIPTION
**Pull request by Asifrizwan6989**
Signin and signup pages

### CODE REVIEW

File: autozone-poc/src/components/Signin/Signin.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const [password, setPassword] = useState("");
+
+
+  const onSubmit = (evt) => {
+    const data = {};
+    evt.preventDefault();
+    data.emailid = emailid;
+    data.password = password;
+    axios.post("http://localhost:5000/user/signin", data).then((response) => {
+      console.log(response);
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Signin/Signin.js]: remove console.log

File: autozone-poc/src/components/Signin/Signin.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      localStorage.setItem('accessToken', response.data.accessToken)
+      window.location.assign("/");
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Signin/Signin.js:21]: can you use a router redirect

File: autozone-poc/src/components/Signup/Signup.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+  export default function Signup (props) {
+  // const {register, handleSubmit, errors, watch}=useForm();
+  const [firstname, setFirstName] = useState("");
+  const [lastname, setLastName] = useState("");
+  const [emailid, setEmailid] = useState("");
+  const [password, setPasswords] = useState("");
+  const [confirmpassword, setConfirmPassword] = useState("");
+
+  $(document).ready(function(){
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Signup/Signup.js:18]: remove jquery validation and implement state-based validation in react

File: autozone-poc/src/components/Signup/Signup.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  });
+
+  const onSubmit = (evt) => {
+    const data = {};
+    evt.preventDefault();
+    data.firstname = firstname;
+    data.lastname = lastname;
+    data.emailid = emailid;
+    data.password = password;
+    axios.post("http://localhost:5000/user/signup", data).then((response) => {
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Signup/Signup.js:39]: dont use hard-coded links

**Review by gokulwebdeveloper — CHANGES_REQUESTED** [review]

File: autozone-poc/src/components/Signup/Signup.js (hunk 3 above)
Comment (Asifrizwan6989) [reply autozone-poc/src/components/Signup/Signup.js:18]: Validation already written in j-query for reusable component, that why i written in j-query to achieve scenario

File: autozone-poc/src/components/Signup/Signup.js (hunk 4 above)
Comment (Asifrizwan6989) [reply autozone-poc/src/components/Signup/Signup.js:39]: the code will work with that url:http://localhost:5000/user/signup

File: autozone-poc/src/components/Signin/Signin.js (hunk 1 above)
Comment (Asifrizwan6989) [reply autozone-poc/src/components/Signin/Signin.js]: removed

File: autozone-poc/src/components/Signin/Signin.js (hunk 2 above)
Comment (Asifrizwan6989) [reply autozone-poc/src/components/Signin/Signin.js:21]: i will implement in next version

**Review by gokulwebdeveloper — APPROVED** [review]

**Pull request merged**